test/node/extended_json.test.ts

```diff
@@ -574,6 +574,70 @@ describe('Extended JSON', function () {
     expect(result).to.deep.equal({ a: 1 });
   });
 
+  describe('ignoreUndefined option', () => {
```

Comment:
The changes you've made implicitly add support for `ignoreUndefined` to our `EJSON.serialize` function too.  Could you add a set of tests like these tests for the `EJSON.serialize` function as well?